src/components/SelectTrackModal.tsx

```
@@ -179,27 +276,32 @@ export function SelectTrackModal({
                           src={track.artwork_url}
                           alt={track.track_name}
                           fill
                           className="object-cover"
                           unoptimized
                         />
+                        {playingId === trackId && (
+                          <div className="absolute inset-0 flex items-center justify-center bg-black/40 backdrop-blur-[2px]">
+                            <Pause className="h-5 w-5 text-white fill-current" />
+                          </div>
+                        )}
                       </div>
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)